### PR TITLE
[M-FIX-1656]

### PR DESCRIPTION
**Pull request by JoseSolerccxc**
## Related Issue
redmine #issue | Path | Description
--- | --- | --- |
<a target='_blank' href='http://redmine.ccxc.co/issues/1656'>#1656 </a> | <a href='http://localhost:3000'>http://localhost:3000 </a> |  comentarios |

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
```
Comment (hevillalobosccxc) [inline public/index.html:28]: Validar inputs
Screen Shot 2022-09-22 at 5 10 52 PM

File: public/index.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
-    <title>DiggiPymes</title>
+    <title>diggi pymes</title>
```
Comment (hevillalobosccxc) [inline public/index.html:30]: Separar la palabra gratis y acomodar el puntero
Screen Shot 2022-09-22 at 5 12 03 PM

**Review by hevillalobosccxc — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (JoseSolerccxc) [reply public/index.html:28]: Done

File: public/index.html (hunk 2 above)
Comment (JoseSolerccxc) [reply public/index.html:30]: Done

**Review by hevillalobosccxc — APPROVED** [review]

**Pull request merged**